### ISSUE TITLE
Using MathJax

### ISSUE THREAD

**Issue by nadr0** [parkjs814/AlgorithmVisualizer#207]
I think it would be useful to have Latex renderings of the space and time complexities when describing the algorithms. Right now its just being rendered with HTML tags. We could potentially use [MathJax - Github](https://github.com/mathjax/MathJax), [MathJax - Website](https://www.mathjax.org/). 

The library would provide a cleaner rendering of the typical asymptotic notations. Also the creator would be able to provide proofs or more deeper explanations of space and time complexities. 

Thoughts? 

Edit: PR #209

**Comment by parkjs814** [comment]
That would be helpful not only for time complexities but also for other description of the algorithms.

**Comment by TornjV** [comment]
#209

**Comment by nem035** [comment]
Just wondering, would it be possible to build/compile formula images in advance and use those?
The rendering delay is pretty obvious right now

**Comment by nadr0** [comment]
Hey I took a look into this, because I am using CDN to request the render of the mathjax it can take awhile to load. If we include the mathjax library directly in the repo it will load everything faster. Also the renders of Mathjax do get cached (both with CDN and local library). The library is about 19.4MBs and its a decent amount of files. 

If we are going to be rendering more Mathjax on the site other than the runtimes we probably should use the library. 

I tried this locally and its super fast render times.
